tools/flow9/backends/cpp/cpp.flow

```diff
@@ -445,3 +482,42 @@ switchType2caseIndexes(switchType : BType, backend : Backend<CppInfo>) -> Tree<s
 		BTypeFunction(__, __) : makeTree(); // error
 	}
 }
+
+isBTypeUnionType(type : BType, backend : Backend<CppInfo>) -> bool {
```

Comment:
Why not just pass the CppInfo directly? Alternatively, move this to another file, in case it is more generally useful, and then let Backend<?> be polymorphic.